setup.py

```diff
@@ -35,7 +35,7 @@
 ]
 PACKAGES_ALL_API = [
     "anthropic>=0.18",
-    "cohere",
+    "cohere<5.0.0a0",
```

Comment:
We are importing class `cohere.CohereError` here:

https://github.com/tatsu-lab/alpaca_eval/blob/f5046aeac0a6d5ea6a665bf2bbf7c8898c1d30ed/src/alpaca_eval/decoders/cohere.py#L10

The `CohereError` was removed in `cohere` v5, which was released yesterday ([release history on PyPI](https://pypi.org/project/cohere/#history)).

`cohere.CohereError` in cohere v4: https://github.com/cohere-ai/cohere-python/blob/v4/cohere/__init__.py#L5
`cohere.*Error` in cohere v5: https://github.com/cohere-ai/cohere-python/blob/67620c348329308186d0b7e771a06795ea718226/src/cohere/__init__.py#L122-L130

Reply:
That makes sense, thanks!